client/jetpack-connect/jetpack-new-site/index.jsx

```diff
@@ -58,77 +46,43 @@ class JetpackNewSite extends Component {
 			<div>
 				<BackButton onClick={ this.handleBack } />
 				<div className="jetpack-new-site__main jetpack-new-site">
-					<div className="jetpack-new-site__header">
-						<h2 className="jetpack-new-site__header-title">
+					<header className="formatted-header">
+						<h1 className="formatted-header__title">
 							{ this.props.translate( 'Add a New Site' ) }
-						</h2>
-						<div className="jetpack-new-site__header-text">
-							{ this.props.translate(
-								'Create a new site on WordPress.com or add your existing self-hosted WordPress site with Jetpack.'
-							) }{' '}
-						</div>
-					</div>
+						</h1>
+						<p className="formatted-header__subtitle">
+							{ this.props.translate( 'Create a new site or add your existing self-hosted site.' ) }{' '}
+						</p>
+					</header>
 					<div className="jetpack-new-site__content">
-						<Card className="jetpack-new-site__wpcom-site">
-							<WordPressLogo />
-							<h3 className="jetpack-new-site__card-title">
-								{ this.props.translate( 'Create a shiny new WordPress.com site' ) }
-							</h3>
-							<div className="jetpack-new-site__card-description">
-								<p>
-									{ this.props.translate(
-										"Tell us what type of site you need and we'll get you setup. " +
-											'If you need help we’ve got you covered with 24/7 support.'
-									) }
-								</p>
-							</div>
-							<div className="jetpack-new-site__button-holder">
-								<Button
-									className="jetpack-new-site__button button is-primary"
-									href={ this.getNewWpcomSiteUrl() }
-								>
-									{ this.props.translate( 'Start Now' ) }
-								</Button>
-							</div>
-						</Card>
-						<Card className="jetpack-new-site__jetpack-site">
-							<JetpackLogo size={ 72 } />
-							<h3 className="jetpack-new-site__card-title">
-								{ this.props.translate( 'Add an existing WordPress site with Jetpack' ) }
-							</h3>
+						<Card className="jetpack-new-site__card">
 							<div className="jetpack-new-site__card-description">
-								{ this.props.translate(
-									'We’ll be using the Jetpack plugin to connect your site to WordPress.com.'
-								) }
-							</div>
-							<SiteUrlInput
-								onChange={ this.handleJetpackUrlChange }
-								onSubmit={ this.handleJetpackSubmit }
-								handleOnClickTos={ this.handleOnClickTos }
-								url={ this.state.jetpackUrl }
-								autoFocus={ false } // eslint-disable-line jsx-a11y/no-autofocus
-							/>
-						</Card>
-						<Card className="jetpack-new-site__mobile">
-							<div className="jetpack-new-site__mobile-wpcom-site">
-								<p>{ this.props.translate( 'Create a new shiny WordPress.com site:' ) }</p>
-								<Button primary href={ this.getNewWpcomSiteUrl() }>
-									{ this.props.translate( 'Start Now' ) }
-								</Button>
+								<p>{ this.props.translate( 'Create a shiny new WordPress.com site.' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Create a shiny new WordPress.com site' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/simplify-add-new-site)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).